DependencyInjection/EzSystemsEzSupportToolsExtension.php

```diff
@@ -15,6 +15,16 @@
 
 class EzSystemsEzSupportToolsExtension extends Extension
 {
+    public function getAlias()
+    {
+        return 'ez_support_tools';
+    }
+
+    public function getConfiguration(array $config, ContainerBuilder $container)
+    {
+        return new Configuration();
```

Comment:
Class is missing. I'm assuming this is still to be implemented. Pointing it out just in case ;)

Reply:
Not anymore :)